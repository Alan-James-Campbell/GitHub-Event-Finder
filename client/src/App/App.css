.App {
  text-align: center;
}

.App-header {
  background-color: CornflowerBlue;
  min-height: 100vh;
  margin: 0 auto;
  text-align: center;
  color: white;
}

