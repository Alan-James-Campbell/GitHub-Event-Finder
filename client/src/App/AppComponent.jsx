import React 			  from 'react'
import EventForm          from '../EventForm'
import                   './App.css'

const App = ({getEvents}) => {

  return (
    <div className="App">
      <header className="App-header">
        <h1>Search For GitHub Events</h1>
        <EventForm/>
      </header>
    </div>
  )
  
}

export default App

