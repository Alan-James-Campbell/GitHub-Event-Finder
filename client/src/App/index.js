import AppComponent  from './AppComponent'
import {connect}     from 'react-redux'
import {fetchEvents} from '../reducers/events'

const mapStateToProps = state => {
  console.log('state', state)
  return {
  
  }
}

const mapDisptachToProps = (dispatch,ownProps) => {
  return {
     getEvents(userName, repoName) {
       dispatch(fetchEvents(userName, repoName))
    }

  }
}

export default connect(mapStateToProps, mapDisptachToProps )(AppComponent)
