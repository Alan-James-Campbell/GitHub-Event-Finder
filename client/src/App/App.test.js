import React       from 'react'
import { shallow } from 'enzyme'
import toJson      from 'enzyme-to-json'
import App         from './AppComponent'
    
describe('App', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<App/>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})