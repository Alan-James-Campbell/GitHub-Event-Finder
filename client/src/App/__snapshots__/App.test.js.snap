// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1>
      Search For GitHub Events
    </h1>
    <Connect(ReduxForm) />
  </header>
</div>
`;
